wasm/demo/src/advanced-params.ts

```diff
@@ -0,0 +1,67 @@
+import { Message } from "didcomm-js";
+import {
+  ALICE_DID,
+  ALICE_DID_DOC,
+  ALICE_SECRETS,
+  BOB_DID,
+  BOB_DID_DOC,
+  BOB_SECRETS,
+} from "./test-vectors";
```

Comment:
I think we don't need 2 test vectors. If we import "../../tests-js/src/test-vectors" then at lest drop them from current folder.